xrpl/core/binarycodec/definitions/definitions.json

```diff
@@ -2974,6 +3056,9 @@
     "DIDDelete": 50,
     "OracleSet": 51,
     "OracleDelete": 52,
+    "CredentialCreate": 54,
+    "CredentialAccept": 55,
+    "CredentialDelete": 56,
```

Comment:
These values are out of date. In the version that's currently available on Devnet, 54-56 are MPT transactors and the Credential transactors are:
```
      "CredentialAccept": 59,
      "CredentialCreate": 58,
      "CredentialDelete": 60,
```

With the values as-is, trying to submit one of these transactions fails with an `invalidTransaction` error such as `Field 'Subject' found in disallowed location.`

Reply:
Thanks, I've updated the file here: [f58dc2c](https://github.com/XRPLF/xrpl-py/pull/759/commits/f58dc2cda825d03b758f4acbc6da55373ebb515f)

I haven't been able to get the CI to use the new docker images. I'll do that on monday and validate the changes.

But if you need the updated definitions, I believe the current version is valid.

Reply:
Also, I have manually edited the fields in the `definitions.json` file. I haven't updated the scripts which auto-generate the `definitions.json` file. Consequently, the serialization pertaining to MPT-related fields do not work in the current version.

I'll need to fix the scripts (or) the authors of the MPT fields will need to manually insert the relevant fields/values into the `definitions.json` file. `server_definitions` is not very helpful because its formatting is different from `definitions.json` file.